cmd/jag/commands/proxy_network.go

```diff
@@ -214,7 +214,7 @@ func runProxyServer(ud *uartDevice, identity *uartIdentity) error {
 	})
 
 	// TODO(florian): this print should be a log.
-	fmt.Printf("Proxying device through 'http://%s:%d'.\n", localIP, localPort)
+	fmt.Printf("Proxying device %s through 'http://%s:%d'.\n", identity.Name, localIP, localPort)
```

Comment:
Maybe we should make this look more like the output you get from the device? It looks like app output as is.

https://github.com/toitlang/jaguar/blob/main/src/network.toit#L45

Reply:
Not sure.
I would almost like to make the monitor output less like the output from the device.

Reply:
Then you'll have three things you have to distinguish.